docker-compose.yml

```diff
@@ -0,0 +1,71 @@
+version: "3"
+services:
+  editor:
+    build:
+      context: .
+    ports:
+      - 8000:8000
+    depends_on:
+      - pipeline
+  pipeline:
```

Comment:
There's a crap-ton of copy-pasta here that's from the pipeline's docker-compose config. I don't know if there's a better way to declare dependencies so that we don't need all this identical config in both repos. Advice requested.

Reply:
I'm not sure either or what the best practices are in this case. In the past, I've tried to share base_container between repos but some of the container configuration info the docker-compose.yml is going to be duplicative.